client/components/date-range/inputs.tsx

```diff
@@ -0,0 +1,138 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent, useRef, useCallback } from 'react';
+import { noop } from 'lodash';
+import uuidv4 from 'uuid/v4';
+import { localize, LocalizeProps } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { useLocalizedMoment } from 'components/localized-moment';
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import SharedProps from './shared-props';
+
+type StartOrEnd = 'Start' | 'End';
+
+interface Props {
+	startDateValue: string;
+	endDateValue: string;
+	startLabel: string | null | undefined;
+	endLabel: string | null | undefined;
+	onInputFocus: ( value: string | null | undefined, startOrEnd: StartOrEnd ) => void;
+	onInputBlur: ( value: string | null | undefined, startOrEnd: StartOrEnd ) => void;
+	onInputChange: ( value: string | null | undefined, startOrEnd: StartOrEnd ) => void;
+}
+
+export const DateRangeInputs: FunctionComponent< Props & SharedProps & LocalizeProps > = props => {
```

Comment:
Same here - as a `FunctionComponent` shall we use `useTranslate` hook?

```suggestion
export const DateRangeInputs: FunctionComponent< Props & SharedProps > = props => {
```

Reply:
Done!